sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+Write-Warning "Certificates created by this script MUST NOT be used for production."
+Write-Warning "They expire after 30 days, and most importantly are provided for demonstration purposes to help you quickly understand CA Certificates."
+Write-Warning "When productizing against CA Certificates, you'll need to use your own security best practices for certification creation and lifetime management."
+
+# Check to make sure openssl is installed
+if (-Not (Get-Command "openssl" -ErrorAction SilentlyContinue)) {
+  throw "openssl is not availabe. You will need to install it or add it to the PATH to proceed"
+}
+
+# Remove previous cert stores
+if (Test-Path device_cert_store.pem) {
+  Remove-Item device_cert_store.pem
+}
+
+#Wrap commands to catch errors
+function Invoke-IoTCommand {
+
+  param(
+    [string]$Command = $(throw "Must provide executable command string")
+  )
+
+  Invoke-Expression "$Command"
+
+  if ($LastExitCode -ne 0) {
+    throw "'$Command' invocation failed with exit code ($LastExitCode)"
+  }
+}
+
+Invoke-IoTCommand -Command 'openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey'
+Invoke-IoTCommand -Command 'openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"'
+Invoke-IoTCommand -Command 'openssl x509 -noout -text -in device_ec_cert.pem'
+
+Get-Content device_ec_cert.pem, device_ec_key.pem | Set-Content device_cert_store.pem
+
+if ($IsWindows -or $IsMacOS) {
+  Write-Output "`n"
+  Write-Warning "IMPORTANT:"
+  Write-Warning "It is NOT recommended to use OpenSSL on Windows or OSX. Recommended TLS stacks are:"
+  Write-Warning "Microsoft Windows SChannel: https://docs.microsoft.com/en-us/windows/win32/com/schannel"
```

Comment:
Pretty sure it's `Schannel`, not `SChannel`.  E.G. https://docs.microsoft.com/en-us/windows/win32/com/schannel